requirements.txt

```diff
@@ -3,6 +3,7 @@ flake8
 # rst_include tool as GitHub doesn't support `.. include::` when rendering
 # previews.
 rst_include
+pandas
```

Comment:
pandas seems a rather heavy dependency here? As pandas is a binary dependency it should probably also be in the `environment.yml`?

Reply:
I used pandas to simplify accessing row fields. If pandas is too heavy I can replace it.

Reply:
I put `pandas` in `requirements.txt` in case someone would like to install the required `pip` packages for skywater-pdk without using `conda` environment.

Reply:
I also used pandas to read Google spreadsheet files directly from URL